### PR TITLE
Polygon copy updates

### PR DESCRIPTION
**Pull request by zacheryschiller**
##### Description

Updating some of the Polygon copy to read better.

##### Testing

Tested locally, but needed to bump the ipdb version [as in this PR](https://github.com/gitcoinco/web/pull/9443) to run Docker locally.

### CODE REVIEW

**Review by gcc20 — APPROVED** [review]

**Review by chibie — APPROVED** [review]

File: requirements/base.txt [hunk 1]
```diff
@@ -49,7 +49,7 @@ django-ipware==2.0.2
 geoip2==2.8.0
 django-silk==2.0.0
 django-extensions==2.2.1
-ipdb==0.11
+ipdb==0.13.9
```
Comment (thelostone-mc) [inline requirements/base.txt:52]: why are we bumping this here ? Could this be done in it's own PR ?

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
copy changes LGTM
Left a comment on unrelated change

File: requirements/base.txt (hunk 1 above)
Comment (zacheryschiller) [reply requirements/base.txt]: You actually already merged this in another PR: https://github.com/gitcoinco/web/pull/9443 :-) But I updated the base so we're good now.

**Comment by zacheryschiller** [comment]
@thelostone-mc Squashed and updated base branch. Ready to go.

**Review by octavioamu — APPROVED** [review]
